PontoCom/View/LoginView.swift

```diff
@@ -61,11 +61,11 @@ struct LoginView: View {
             .padding(.vertical, 20)
         }
         .padding()
-        .onChange(of: viewModel.isAuthenticated) { oldValue, newValue in
-            if newValue {
-                isAuthenticated = true
-            }
-        }
+//        .onChange(of: viewModel.isAuthenticated) { oldValue, newValue in
+//            if newValue {
+//                isAuthenticated = true
+//            }
+//        }
```

Comment:
Descomentar essas linhas

```suggestion
       .onChange(of: viewModel.isAuthenticated) { oldValue, newValue in
            if newValue {
                isAuthenticated = true
            }
        }
 ```